lib/trino-filesystem/src/main/java/io/trino/filesystem/hdfs/HdfsFileSystem.java

```diff
@@ -125,4 +157,12 @@ public FileIO toFileIo()
     {
         return new ForwardingFileIo(this);
     }
+
+    private static FileSystem getRawFileSystem(FileSystem fileSystem)
```

Comment:
Would it make sense to move, as a prep commit, `io.trino.plugin.hive.util.HiveWriteUtils#getRawFileSystem` over here to this module?

Reply:
It is already there but trino-filesystem doesn't have trino-hive as a dependency. And IMHO it should not have, am I wrong ?